src/UI/Component/Symbol/Glyph/Factory.php

```diff
@@ -526,7 +521,7 @@ public function comment(string $action = null): Glyph;
 	 *          A Status Counter MUST indicate the overall amount of like expressions.
 	 *   accessibility:
 	 *       1: >
-	 *          The aria-label MUST be 'Like'.
+	 *          The aria-label MUST be ‘Like' for the first (inactive) version of the glyph and ‘Undo Like' for the second (active) version.
```

Comment:
There is a whole set of such accessibility rule change for the aria-label. I like them. 

They need JF decision, and they will also need some additional implementation. Make a separate PR in which you pack all those aria-rule changes, which we then propose to the JF.